cmd/chihaya/main.go

```diff
@@ -73,7 +73,7 @@ func (r *Run) Start(ps storage.PeerStore) error {
 
 	if cfg.HTTPConfig.Addr != "" {
 		log.Info("starting HTTP frontend", cfg.HTTPConfig.LogFields())
-		httpfe, err := http.NewFrontend(r.logic, cfg.HTTPConfig)
+		httpfe, err := http.NewFrontend(r.logic, &cfg.RequestSanitizer, cfg.HTTPConfig)
```

Comment:
Do we maybe need to check if `cfg.RequestSanitizer` is set or left empty and use defaults?